docs/src/design/stack/crypto_ops.md

```diff
@@ -119,3 +119,34 @@ The above constraint enforces that the specified input and output rows for both,
 
 The effect of this operation on the rest of the stack is:
 * **No change** for positions starting from $4$.
+
+## FRIE2F4
+The `FRIE2F4` operation performs FRI layer folding by a factor of 4 for FRI protocol executed in a degree 2 extension of the base field. It also performs several computations needed for checking correctness of the folding from the previous layer as well as simplifying folding of the next FRI layer.
+
+The stack for the operation is expected to be arranged as follows:
+- The first $8$ stack elements contain $4$ query points to be folded. Each point is represented by two field elements because points to be folded are in the extension field. We denote these points as $q_0 = (v_0, v_1)$, $q_1 = (v_2, v_3)$, $q_2 = (v_4, v_5)$, $q_3 = (v_6, v_7)$.
+- The next element $f\_pos$ is the query position in the folded domain. It can be computed as $pos \mod n$, where $pos$ is the position in the source domain, and $n$ is size of the folded domain.
```

Comment:
Do you mean $f_{pos}$ ? If so, surround `pos` with `{}`

Reply:
I actually did want an underscore there - `f_pos` as it stands for "folded position". $f_{pos}$ doesn't quite work. I guess we could change it to $pos_{folded}$ but it is a bit more verbose.

Reply:
Oh I misunderstood. Makes sense now.